infrastructure/helm-chart/charts/apps/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for the Airy Core Platform application
-name: core
-version: 0.5.0
+name: apps
+version: 1.0
```

Comment:
`0.1.0` didn't mean much sense I think, so I put `1.0` as this is required, so that at least it is the same as the `apiVersion`